test/navbar-components/navigationBarTest.js

```diff
@@ -379,10 +379,8 @@ describe('navigationBar tests', function () {
           this.app.client
             .activateURLMode()
             .waitForExist(urlBarIcon)
-
-            // TODO: Fix
             .getAttribute(urlBarIcon, 'class').then((classes) =>
-              classes.includes('fa-unlock') && classes.includes('insecure-color')
+              classes.includes('fa-unlock')
```

Comment:
Can we do it in this one? Because we changes logic of how color is added I think this test should be fixed in this PR

Follow ups just add issues and PR's to the already long QA testing list :smiley: